adb/device-manager.js

```diff
@@ -47,6 +49,13 @@ export class DeviceManager {
       this.onDeviceFeaturesUpdated(device);
     });
 
+    ipcRenderer.on('portscan-requested', (event, args) => {
+      args = (args !== undefined ? args : {});
+      let ipAddress = (args.ip !== undefined ? args.ip : ip.address());
```

Comment:
I would shorten it like this `let ipAddress =  args.ip || ip.address();`. This should work, if ip is a string, as I would assume.